### ISSUE TITLE
Any good and profitable bot to share? It seems I'm not finding the right bot.

### ISSUE THREAD

**Issue by KingSo7** [binary-com/binary-bot#3533]
I'm always losing with any bot that I create or get. Can anyone recommend a good bot for small account please. Thank you in advance

**Comment by bobmeg** [comment]
Just learn the markets...thats the best advice...there are loads of chances for profit...don expect someone genuin to share with you their edge in trading...you have to find your own edge

**Comment by binary-ex-machina** [comment]
TL/DR: Use indicators to avoid big losing streaks. That's the best you can do to survive this game, and use martingale (my experience and my thoughts )

Now to the more detailed explanation

@KingSo7 I saw this on another issue, but I. couldn't find it. Take any strategy that uses the trade type rise/fall as example. You have 50% of the chances of being right when picking what trade type to go. Using one indicator or ten will give you the same win rate over time (I am talking about my tests with 10.000 trades, minimum). 

So you may ask: playing the bot with any strategy will give the same result? Yes, but after many trades (10k again), you will see the win rate is the same. 

Why? Because binary options have an expiry time. You can't do anything about it. It's how the game works.

What can you do to bend the rules in your favor? Sell the contract before expiry, or trade with a massive account, use martingale and pray that it will not blow your budget (yes, $200k balance, and you still can lose all. it just needs a choppy market or a bad strategy, and boom, it's gone). 

You can try to use indicators to avoid big losing streaks (what I would choose from the above options), which is what will
destroy your account depending on your money management (martingale and likes). 

**The riddle is this one (based on all my tests from the past four years of making bots and strategies for Derv/Binary)**
If your account can sustain a profit per day enough to recover from the expected stop losses you will get during that same period, you can make money on this market. Otherwise, it's just luck.

You will make an entire month of profit, then you will lose in the subsequent gain, and that doesn't happen because Deriv is cheating or trying to steal your money. It's just like the odds are when trading binary options. 

If your bot gets eight losses in a row before 500 trades, you can still improve—ten losses before 100 trades. So there is space to make it better. Only testing and backtesting will answer what to use, But I will get rid of right from the start indicators like RSI, MACD, Stoch, and others like them. It's just too many false signals. It will make It worse.

Always trade with the trend, so Moving averages are my indicators, with others for possible reversals. I hope it helps anyone here improve how you search for a strategy and use it on your account.

**Comment by bobmeg** [comment]
Stop running bots to seek profit....stop being lazy and watch the markets to look for the most opportune moments in common market structure patterns...then run your bot for moment alone...that way...i can garantee you will earn good small small profit even from a low balance account...its all about timing..not just allowing a bot to run

**Comment by KingSo7** [comment]
Wow @binary-ex-machina that's captivating. Thank you very much. I will try the indicator aspect too.

**Comment by binary-ex-machina** [comment]
@VERTO13 In my trading, I use moving averages. Two simple moving averages, one with 20 periods and another with 200. When both MAs are aligned (MA20 above MA200 and going up, or MA20 below MA200 and going down), I make trades

When both are increasing, MA20 above 200, I take Buy trades. When both are going down, MA20 below MA200, I take sell trades.

There are many indicators you can use to avoid losing streaks, but going with the trend works best for me.

**Comment by kibomamoses** [comment]
@binary-ex-machina  what is your time period between the candles and how many minutes do you use before that purchase or sale is completed?
